### PR TITLE
Tickets/dm 30114

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: bin/build_bush_cycle.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (mareuter) [inline bin/build_bush_cycle.sh:1]: I think this script should be called `build_push_cycle.sh`. :-)

File: build/love/manager-static/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Clone repo
+FROM alpine/git as cloner
+
+WORKDIR /usr/src/love
+ARG love_manager
+RUN git clone --branch ${love_manager} --depth 1 --single-branch https://github.com/lsst-ts/LOVE-manager.git
+
+FROM python:3.8.2-buster as builder
+COPY --from=cloner /usr/src/love/LOVE-manager /usr/src/love/LOVE-manager
```
Comment (mareuter) [inline build/love/manager-static/Dockerfile:9]: This is a general comment that applies to all of the directories specified like this in both manager-static and manager Dockerfiles. This seems overly redundant. Is there something in the package reorg that necessitated changing this?

File: cycle/docker-compose.yaml [hunk 3]
```diff
@@ -624,6 +679,14 @@ services:
     image: ${hub}/lsst_sims:${CYCLE}
     <<: *base-lsst_sims
 
+  adam-sensors:
+    image: ${hub}/adam-sensors:${CYCLE}
+    <<: *base-adam-sensors
+
+  adam-sensors_c:
```
Comment (mareuter) [inline cycle/docker-compose.yaml:686]: Looks like this is backwards from the other specs. This one should be without the `_c` and the one above should have the `_c`.

File: build/love/frontend/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Clone repo
+FROM alpine/git as cloner
+
+WORKDIR /usr/src/love
+ARG love_frontend
+RUN git clone --branch ${love_frontend} --depth 1 --single-branch https://github.com/lsst-ts/LOVE-frontend.git
+
+# Build artifacts
+FROM node:12.16.1-alpine as builder
+COPY --from=cloner /usr/src/love/LOVE-frontend /usr/src/love/LOVE-frontend
```
Comment (mareuter) [inline build/love/frontend/Dockerfile:10]: Missed this one earlier. Falls in the same vein as the comments about the manager Dockerfiles.

**Review by mareuter — CHANGES_REQUESTED** [review]
One small fix needed, but I want to really understand what necessitated the directory changes in the LOVE Dockerfiles.

File: build/love/manager-static/Dockerfile (hunk 2 above)
Comment (tribeiro) [reply build/love/manager-static/Dockerfile:9]: You mean using a container to clone and them copy it over? If yes, then the problem is that `python:3.8.2-buster` does not contain git, so I can't really clone the repo. I thought using this construct was better than installing git.

Comment (mareuter) [reply build/love/manager-static/Dockerfile:9]: No. I meant the `/usr/src/love/LOVE-manager`. Seems like `/usr/src/love/manager` is much simpler.

**Review by mareuter — APPROVED** [review]
Thanks for the fixes. This looks fine now.

**Pull request merged**